src/tests/components/experiments/ExperimentList.test.js

```diff
@@ -48,3 +56,44 @@ describe("Experiment list component", () => {
     );
   });
 });
+
+describe("Test renderTitle() method", () => {
+  it("should render a title showing 1 experiment filtered, and the date range that it falls between", () => {
+    const wrapper = shallow(
+      <ExperimentList experimentCount={113} 
+        fetchExperiments={() => {}} 
+        filteredExperiments={fromJS([{}])}
+        startDate={"2018-08-22"}
+        endDate={"2019-01-20"}
+      />
+    );
+    const text = wrapper.find("h1").text();
+    expect(text).toEqual("1 / 113 experiments after 2018-08-22 before 2019-01-20");
```

Comment:
Very nice 👍  I just want to point out that the more you can hard code the expected output of something in a test, the better.